src/main/mondrian/util/CancellationChecker.java

```diff
@@ -0,0 +1,39 @@
+/*
+// This software is subject to the terms of the Eclipse Public License v1.0
+// Agreement, available at the following URL:
+// http://www.eclipse.org/legal/epl-v10.html.
+// You must accept the terms of that agreement to use this software.
+//
+// Copyright (C) 2002-2005 Julian Hyde
+// Copyright (C) 2005-2016 Pentaho and others
```

Comment:
New files should have copyright years set as :

Copyright (C) 2016-2016 Pentaho and others